packages/db/src/main/cfxjs/db/queries.cljs

```diff
@@ -971,7 +971,7 @@
                           :token/name     name
                           :token/symbol   symbol
                           :token/decimals decimals
-                          :token/logoURI  image
+                          :token/logoURI  (if (nil? image) "" image)
```

Comment:
better change it this way
![image](https://user-images.githubusercontent.com/15090582/164579466-b263e748-56e5-4217-bcd4-91a30573c176.png)

Reply:
:sunflower::sunflower::sunflower::smiley::smiley::smiley:
Fixed and have already tested it on local.
Thanks for this info. Learned a new knowledge [assoc API of Clojure](https://clojuredocs.org/clojure.core/assoc)